app/models/genre.rb

```diff
@@ -0,0 +1,6 @@
+class Genre < ApplicationRecord
+ validates :name, presence: true, exclusion: { in: [nil] }
```

Comment:
`presence: true` is already validating that the name is not nil OR an empty string, so you can remove the `exclusion` validation here!